firebase.json

```diff
@@ -20,6 +20,14 @@
       "source" : "/latest-debug.min.js",
       "destination" : "/client/v1.0.1-rc1-debug.min.js",
       "type" : 302
```

Comment:
Should we remove this rc1 code here?

Reply:
Nah, we remove that when it goes live. it's in "beta" right now, below.

Reply:
Good thought though

Reply:
👍